src/Kale.hs

```diff
@@ -50,6 +50,18 @@ data Task = Task
     }
     deriving (Eq, Show)
 
+type Qualified = String
+type Module = String
+
+data Import = Import Module (Maybe Qualified)
+
+data Syntax = Syntax
+    { imports :: [Import]
+    , commandSumType :: CommandSumType
+    , driver_ :: Driver
+    --      ^ underscore to avoid conflict with the function 'driver'.
+    }
```

Comment:
This is close! We would definitely want `imports`, but `CommandSumType` is a little higher level than what we want. Ultimately, I think this should be a *very* simple description of Haskell syntax, so something like:

```haskell
data Syntax = Syntax
    { imports :: [Import]
    , dataDecl :: DataDecl
    , caseExpr :: CaseExpr
    -- ...

data DataDecl = SumData [(Constructor, [Args])]
``` 

along with functions to render the `Syntax` as a string.

Reply:
Ah! Okay, I'll try to incorporate these.

Reply:
Oops! Forgot that the current PR gets updated! Should've worked on a different branch.